atc/db/build.go

```diff
@@ -447,32 +446,30 @@ func (b *build) Job() (Job, bool, error) {
 	return job, true, nil
 }
 
-func (b *build) SetComment(comment string) error {
-	var nullComment sql.NullString
-	if comment != "" {
-		nullComment = sql.NullString{String: comment, Valid: true}
-	}
+func (b *build) Comment() (string, error) {
+	var comment string
+	err := psql.Select("COALESCE(bc.comment, '')").
```

Comment:
I'm not 100% sure, but I'm guessing that making this query for each build will end up being more expensive than just always performing a `LEFT JOIN` in the `buildsQuery`, since with this we need to do N+1 round trips to the DB. Note that the `buildsQuery` isn't used for things like scheduling, which was my original concern around performance - but it's still a bit silly to fetch comments for builds if we would just end up ignoring them (e.g. when listing team builds).

I think the better approach conceptually would be to only perform the `LEFT JOIN` when we care about the comment, but the way our ORM is currently structured makes that a bit difficult. So, I think I'm happy just adding it into the `buildsQuery`.